ui/src/app/views/settings/worker-model/list/worker-model.list.html

```diff
@@ -18,37 +18,30 @@ <h2>{{ 'worker_model_list_title' | translate }}</h2>
       <table class="ui fixed celled table" *ngIf="getDataForCurrentPage().length > 0">
           <thead>
           <tr>
+              <th class="three wide">{{ 'worker_model_name' | translate }}</th>
+              <th class="two wide">{{ 'worker_model_description' | translate }}</th>
               <th class="one wide">{{ 'worker_model_type' | translate }}</th>
-              <th class="four wide">{{ 'worker_model_name' | translate }}</th>
               <th class="three wide">{{ 'worker_model_image' | translate }}</th>
-              <th class="three wide">{{ 'worker_model_group' | translate }}</th>
+              <th class="two wide">{{ 'worker_model_group' | translate }}</th>
           </tr>
           </thead>
           <tbody>
           <tr *ngFor="let v of getDataForCurrentPage()">
               <td>
                   <a class="ui" [routerLink]="[v.name]">
-                    <div class="ui">{{v.type}}</div>
-                  </a>
-              </td>
-              <td>
-                  <a class="ui" [routerLink]="[v.name]">
-                    <div class="ui">{{v.name}}
-                        <span *ngIf="v.disabled==='true'"> {{ 'worker_model_disabled' | translate }}</span>
+                    <div class="ui">
+                        {{v.name}}
+                        <span *ngIf="v.disabled"> {{ 'worker_model_disabled' | translate }}</span>
                         <span *ngIf="v.nb_spawn_err > 0">  <i class="warning sign icon orange" title="{{ 'worker_model_warning' | translate }}"></i> </span>
```

Comment:
can you use smDirTooltip on v.nb_spawn_err > 0 too? This icon (warning sign) does not have a great icon (not same size) nor a great tooltip.